application/console.rb

```diff
@@ -70,38 +59,155 @@ def load
 
   def create_the_first_account
     puts "there are no active accounts, type 'y' if you want to create one"
-    gets.chomp == 'y' ? create : console
-  end
-
-  def commands
-    {
-        SC: proc { show_cards },
-        CC: proc { create_card },
-        DC: proc { destroy_card },
-        PM: proc { put_money },
-        WM: proc { withdraw_money },
-        SM: proc { send_money },
-        DA: proc { destroy_account; exit }
-    }
+    gets.chomp == YES ? create : console
   end
 
   def main_menu
     loop do
       main_menu_message
       command = gets.chomp
 
-      if commands.key?(command.to_sym)
-        commands[command.to_sym].call
-      elsif command == 'exit'
-        exit
-        break
+      case command
+      when COMMANDS[:show_cards]      then show_cards
+      when COMMANDS[:create_card]     then create_card
+      when COMMANDS[:destroy_card]    then destroy_card
+      when COMMANDS[:put_money]       then put_money
+      when COMMANDS[:withdraw_money]  then withdraw_money
+      when COMMANDS[:send_money]      then send_money
+      when COMMANDS[:destroy_account] then return destroy_account
+      when EXIT                       then break exit
       else
         puts "Wrong command. Try again!\n"
       end
     end
   end
 
+  def show_cards
+    return puts "There is no active cards!\n" unless cards
+
+    @account.current_account.card.each do |card|
```

Comment:
you call `each` on `card`. card should be plural:
`@account.current_account.cards`

Reply:
it was fixed.